src/components/forms/EncryptionKeyForm.js

```diff
@@ -4,7 +4,7 @@ import {
   buildFormFunctions,
 } from "../component-utils";
 import { IcseFormGroup } from "../Utils";
-import { IcseNameInput, IcseToggle } from "../Inputs";
+import { IcseNameInput, IcseToggle, IcseTextInput } from "../Inputs";
```

Comment:
do we need to use the new endpoint dropdown in this form somewhere?

Reply:
yup